src/Features/CSharp/Portable/Completion/CompletionProviders/NamedParameterCompletionProvider.cs

```diff
@@ -79,11 +79,6 @@ internal override bool IsInsertionTrigger(SourceText text, int characterPosition
                 return;
             }
 
-            if (token.IsMandatoryNamedParameterPosition())
-            {
-                context.IsExclusive = true;
```

Comment:
I refined this to keep the old behavior for old language versions.

I'm working on the VB side as well, but I'm having troubles with the baseline test. The `CurrentCompletionPresenterSession` is getting set twice (once with the correct completion values, and a second time with `null`). Still investigating.